lib/engine/game/g_1854/step/waterfall_auction.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/waterfall_auction'
+
+module Engine
+  module Game
+    module G1854
+      module Step
+        class WaterfallAuction < Engine::Step::WaterfallAuction
+          def actions(entity)
+            return [] if @game.need_auction_or
+
+            super
+          end
+
+          def setup
+            super
+            original_companies = @game.initial_auction_companies.sort_by(&:value)
+            @companies = @game.companies.select { |company| !company.owned_by_player? && !company.closed? }
+            @cheapest = original_companies.first
+          end
+
+          def all_passed!
+            # Everyone has passed so we need to run a fake OR.
```

Comment:
Remove misleading and misplaced comment.
```suggestion
```

Reply:
What is misleading about it?

Reply:
It might be better placed right above `@game.trigger_auction_or`